pkg/security/securitycontextconstraints/provider.go

```diff
@@ -269,6 +269,28 @@ func (s *simpleProvider) ValidatePodSecurityContext(pod *api.Pod, fldPath *field
 
 	allErrs = append(allErrs, s.sysctlsStrategy.Validate(pod)...)
 
+	if len(pod.Spec.Volumes) > 0 && len(s.scc.AllowedFlexVolumes) > 0 && sccutil.SCCAllowsFSType(s.scc, securityapi.FSTypeFlexVolume) {
```

Comment:
odd... I expected this to extend the existing volume validation. should that move from ValidateContainerSecurityContext here as well?

Reply:
PR with the aforementioned changes has been submitted: https://github.com/openshift/origin/pull/16006